libyul/backends/wasm/BinaryTransform.cpp

```diff
@@ -336,8 +336,12 @@ bytes BinaryTransform::operator()(BuiltinCall const& _call)
 			_call.functionName.find(".load") != string::npos ||
 			_call.functionName.find(".store") != string::npos
 		)
-			// alignment and offset
-			ret += bytes{{3, 0}};
+			// Alignment and offset hints. Interpreters ignore the alignment. JITs/AOTs can take it
```

Comment:
To precise the offset is not a hint, that needs to be taken into consideration. Only alignment is a hint.

Reply:
Oh, ok. I remembered that the spec said that both were optional so I somehow got into my head that it was a hint too :) Does this mean that if e.g. offset = 5, the `store` happens at ptr+5?

Reply:
Comment updated.

Reply:
> Does this mean that if e.g. offset = 5, the store happens at ptr+5?

Yes